### PR TITLE
feat(web/applications): add webfilter, agent and author metadata template (boas-751)

### PR DESCRIPTION
**Pull request by lutamKC**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/views/applications/case-documentation/documentation-upload.njk [hunk 1]
```diff
@@ -29,7 +29,7 @@
 		formTitle: serviceName,
 		nextPageUrl: 'document-category'|url({caseId:caseId, documentationCategory: currentFolder}),
 		allowedTypes: ['pdf','doc','docx','ppt','pptx','xls','xlsx','jpg','jpeg','mpeg','mp3','mp4','mov','png','tif','tiff' ],
-		disclaimerText: "Saving files will default then to ‘Unredacted’ and ‘Not checked’"
+		disclaimerText: "Saving files will default them to ‘Unredacted’ and ‘Not checked’"
```
Comment (lutamKC) [inline apps/web/src/server/views/applications/case-documentation/documentation-upload.njk:32]: unrelated typo

File: apps/web/src/server/views/applications/case-documentation/edit/max-length-input.njk [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+			class="govuk-input govuk-!-width-two-thirds {{ "govuk-input--error" if error else " " }}"
+			id="{{ layout.metaDataName }}"
+			name="{{ layout.metaDataName }}"
+			type="text"
+			value="{{ value }}"
+			aria-describedby="{{ layout.metaDataName }}-hint"
+			maxlength="{{ layout.maxLength }}">
+	</div>
+
+{% endmacro %}
```
Comment (lutamKC) [inline apps/web/src/server/views/applications/case-documentation/edit/max-length-input.njk:31]: this HTML is identical to the GDS input (markup, classes, aria labels etc...) Only difference is the "maxlength" on the input.

Comment (francescobrenci-AR) [reply apps/web/src/server/views/applications/case-documentation/edit/max-length-input.njk:31]: is it something that you can configure in the GDS component when you import it?

**Review by francescobrenci-AR — APPROVED** [review]

File: apps/web/src/server/views/applications/case-documentation/edit/max-length-input.njk (hunk 2 above)
Comment (lutamKC) [reply apps/web/src/server/views/applications/case-documentation/edit/max-length-input.njk:31]: no. 
there is an alternative GDS component that shows the number of characters while you write but, as like as the regular input, it doesn't allow to set a proper limit (maxlength attribute).

**Pull request merged**